src/main/resources/db/changelog/ddl/0018_EIP1-4481_alter_photo_location_arn_column_in_tables.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="vishal.gupta@valtech.com" id="0018a_EIP1-4481_add_photo_location_arn_to_certificate_table - add photo_location_arn column" context="ddl">
+        <addColumn tableName="certificate">
+            <column name="photo_location_arn" type="varchar(1024)" afterColumn="status">
+                <constraints nullable="false"/>
```

Comment:
Does this need to be `true` until the existing rows are populated?

Reply:
yeah I could mark it as true (nullable) initially and set it back to false (non-nullable) once values are copied over. @sentioservices is our db expert, will see what he thinks?

Reply:
I believe that mysql lets you away with doing this and populates an empty string to satisfy the constraint.

Reply:
Ok 

1. set this new column in `certificate` table to be nullable first, 
2. data copied in next step and column is set back to non-nullable in same changeSet